test/stl.cpp

```diff
@@ -101,3 +102,72 @@ TEST(STL, CROSS_CHECK)
 
     EXPECT_TRUE(binReadObjs[0] == asciiReadObjs[0]);
 }
+
+TEST(STL, PositionsCountMismatch)
```

Comment:
@9prady9 are we going to use UpperCamelCase convention for unit test from now on?

Reply:
> @9prady9 I couldn't figure out why we are getting this: https://travis-ci.org/github/9prady9/MeshIO/jobs/663842229#L3984. It even happened locally when building with `make` only when converage flag is turned on. Any thoughts?
> 
> It didn't happen when I used ninja, probably because it suppressed warnings by default.

I didn't understand what you are talking about ?

Reply:
> @9prady9 are we going to use UpperCamelCase convention for unit test from now on?

I just did it by habit from other project. It is not really that important I guess as long as the name is self-explanatory of what it is testing instead of weird short-cuts and numbers.